zetta_utils/mazepa_layer_processing/common/subchunkable_apply_flow.py

```diff
@@ -0,0 +1,204 @@
+from os import path
+from typing import Any, Callable, Generic, List, Literal, Optional, TypeVar, Union
+
+import attrs
+from torch import Tensor
+from typing_extensions import ParamSpec
+
+from zetta_utils import builder, log, mazepa
+from zetta_utils.bcube import BoundingCube
+from zetta_utils.layer.volumetric import VolumetricIndex, VolumetricLayer
+from zetta_utils.typing import IntVec3D, Vec3D
+
+from ..operation_protocols import BlendableOpProtocol
+from .blendable_apply_flow import BlendableApplyFlowSchema
+from .volumetric_callable_operation import VolumetricCallableOperation
+
+logger = log.get_logger("zetta_utils")
+
+IndexT = TypeVar("IndexT")
+P = ParamSpec("P")
+R_co = TypeVar("R_co", covariant=True)
+T = TypeVar("T")
+
+
+@mazepa.taskable_operation_cls
+@attrs.mutable
+class DelegateBlendOperation(Generic[P, R_co]):
```

Comment:
How about `DelegatedSubchunkedOperation`?

Reply:
Yeah, that sounds good.